packages/p2p/src/components/my-profile/my-profile.jsx

```diff
@@ -24,7 +24,7 @@ const MyProfile = () => {
     if (my_profile_store.error_message) {
         return (
             <div className='my-profile__error'>
-                <Text size='xs' font='loss-danger'>
+                <Text as='p' color='loss-danger' size='xs' className='dp2p-table-error' align='center'>
```

Comment:
Nit, but in P2P, we alphabetize the props, please apply the same wherever you can.